docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '3.8'
+
+services:
+  app:
+    build:
+      context: .
+      args:
+        NODE_VERSION: ${NODE_VERSION:-18}
+    ports:
+      - "${APP_PORT:-3000}:${APP_PORT:-3000}"
```

Comment:
It's better not to get the running port inside the container from an environment variable. I think it should be;

```patch
- - "${APP_PORT:-3000}:${APP_PORT:-3000}"
+ - "${APP_PORT:-3000}:3000"
```

Reply:
This should be fixed as well